src/estimation/include/iDynTree/Estimation/AttitudeQuaternionEKF.h

```diff
@@ -253,6 +253,7 @@ namespace iDynTree
          * @return bool true/false if successful or not
          */
         bool ekfComputeJacobianF(iDynTree::VectorDynSize& x, iDynTree::MatrixDynSize& F) override;
+        bool ekfComputeJacobianF(iDynTree::VectorDynSize& x, iDynTree::VectorDynSize& u, iDynTree::MatrixDynSize& F) override;
```

Comment:
What is `u`? Could it be documented in the doxygen docs? It probably make sense to just document the variant of the method with more arguments, and for the doxygen of the method with less arguments just refer to the method with more arguments using an appropriate doxygen command like http://www.doxygen.nl/manual/commands.html#cmdsa .